moped-editor/src/views/projects/projectView/ProjectActivityEntry.js

```diff
@@ -35,11 +35,19 @@ const ProjectActivityEntry = ({ changeIcon, changeText }) => {
         >
           {
             // maps through the array of objects and applies specified style to the text
-            changeText.map((changeObject, index) => (
-              <span className={classes[changeObject.style]} key={index}>
-                {changeObject.text}
-              </span>
-            ))
+            changeText.map((changeObject, index) =>
+              changeObject.link ? (
+                <span className={classes[changeObject.style]} key={index}>
+                  <Link href={changeObject.link} >
```

Comment:
Instead of the `href` prop here, I think it could be good to use the `component` prop so we can make the link use React Router's Link component. We haven't always been super consistent about this in the past, but it could make a smoother transition to the map.

[Here is how we do it in the project list view](https://github.com/cityofaustin/atd-moped/blob/0f48ef0b5b55cdd7094fcaf205b759b6642825ec/moped-editor/src/views/projects/projectsListView/helpers.js#L248-L253)

Reply:
thank you! I had a vague memory that there was a better way to do this, but couldnt conjure up the memory